demo/timing.c

```diff
@@ -267,23 +267,23 @@ int main(void)
             b.dp[0] |= lbit();
             b.used += 1;
          }
-         mp_sub_d(&a, 1, &c);
+         mp_sub_d(&a, 1uL, &c);
          mp_mod(&b, &c, &b);
-         mp_set(&c, 3);
-         rr = 0;
-         tt = -1;
+         mp_set(&c, 3uL);
+         rr = 0u;
+         tt = 0xFFFFFFFFFFFFFFFFuLL;
```

Comment:
wouldn't `UINT64_MAX` be more appropriate?

Reply:
sure.